docker/php-fpm.Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM php:8.2-fpm
+
+# Install dependencies
+RUN apt-get update && \
+    apt-get install -y libpq-dev hunspell hunspell-en-us libzmq5-dev supervisor nginx gcc git
+
+# Compile and install php-zmq
+#
+# PHP-ZMQ has some ... interesting issues with the Docker container.
+# * There does not appear to be a package for it(?) and it doesn't come installed
+#   with PHP.
+# * The package available on PECL is about 8 years old and the repo is abandoned.
+# So we have to compile it from source. However, this is not that painful. Just
+# annoying that we have to do it ourselves.
+
+# TODO: make this into a script and just copy it over rather than having this
+#       all embedded in the Dockerfile
+# TODO: choose a specific tag or commit to clone from rather than using master,
+#       for reproducible builds
+
+RUN cd /tmp && \
```

Comment:
I was thinking it was really weird that you had to build from source given the ubiquity of php-zmq packages on Ubuntu. But I guess this image is based on Debian. 

Anyway, did you take a look at `https://github.com/mlocati/docker-php-extension-installer`? It claims to support zmq extension in the version table.

Reply:
Worth checking, albeit a 5950x slams through that build in about 4 seconds

Reply:
And managed to get it working though I understand why the previous PR #5 did it from source.